friends/src/App.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import axios from 'axios'
+import Friend from './Friend/Friend';
+import NewFriend from './NewFriend/NewFriend';
+import './App.scss';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      friends: [],
+      postRes: '',
+      friend: {
+        name: '',
+        age: 0,
+        email: ''
+      }
+    }
+  }
+
+  componentDidMount() {
+    axios
+      .get('http://localhost:5000/friends')
+      .then(res => this.setState({
+        friends: res.data
+      }))
+      .catch(err => console.log(err))
+  }
+
+  handleChanges = e => {
+    this.setState({
+      friend: {
+        ...this.state.friend,
+        [e.target.name]: e.target.value
+      }
+    })
+  }
```

Comment:
If you're managing the `friend` state on the `NewList` component, you won't need to clutter App with method that doesn't concern itself, thus keeping your code clean more readable.